crime-application-tracking/build.gradle

```diff
@@ -17,6 +17,7 @@ java {
 
 def versions = [
 		pitest                  : "1.15.3",
+		crimeCommonsVersion    : "2.7.0",
```

Comment:
Worth checking in with the Mod team but I think there is a 2.9.3 version now

Reply:
Done.